native/packages/react-native-bpk-component-card/src/BpkCard.js

```diff
@@ -91,17 +91,23 @@ const BpkCard = (props) => {
   const style = [styles.card];
   if (padded) { style.push(styles.cardPadded); }
   if (focused) { style.push(styles.cardFocused); }
-  if (userStyle) { style.push(userStyle); }
 
   return (
-    <TouchableHighlight
-      accessibilityComponentType="button"
-      underlayColor={colorGray50}
-      style={style}
-      {...rest}
-    >
-      <View style={styles.cardInner}>{children}</View>
-    </TouchableHighlight>
+    <View style={userStyle}>
```

Comment:
The `userStyle` have moved is this a regression?

Reply:
Also might it be confusing that we spread `rest` on the `Touchable`, but add the style to the outer view?

Reply:
Based on all this can you expand on why we need to outer view at all?

Reply:
During standup we agreed to have a container component like the button component does, once that is done I'll update this and explain the problem